service-front/module/Application/src/Contracts/OpgApiServiceInterface.php

```diff
@@ -138,4 +138,12 @@ public function getServiceAvailability(string $uuid = null): array;
      * @return array{decision: string}
      */
     public function requestFraudCheck(string $uuid): array;
+
+    /**
+     * @param string $uuid
+     * @param string $assistance
+     * @param string|null $details
+     * @return void
+     */
+    public function updateCaseAssistance(string $uuid, string $assistance, string $details = null): void;
```

Comment:
Assuming `$assistance` represents the "have you received assistance" question, it should probably be a boolean here and throughout the API.